docs/_articles/en/user-guide/deploy-on-save.md

```diff
@@ -7,15 +7,19 @@ lang: en
 
 Whenever you save a local source file, you can enable immediate deployment of the changes to your default org. You might find this feature useful while trying various user interface and functional behavior or while trying to debug some behavior in an Apex class.
 
-To enable deploy on save, you can:
+To enable deploy on save:
```

Comment:
@sbudhirajadoc - We should mention push.  The setting lets you deploy or push automatically on save.  "Push" is in the setting name, but we aren't mentioning it on the page.

Reply:
Done.